usecase/src/main/java/org/mobilitydata/gtfsvalidator/usecase/ValidateShapeDistTraveled.java

```diff
@@ -71,23 +67,22 @@ public void execute() {
         stopTimePerTripId.forEach(
                 (tripId, stopTimeCollection) -> dataRepo.getStopTimeByTripId(tripId)
                         .forEach((stopSequence, stopTime) -> {
-                            if (stopTime.getShapeDistTraveled() != null) {
+                            final Float shapeDistanceTraveled = stopTime.getShapeDistTraveled();
+                            if (shapeDistanceTraveled != null) {
                                 final Trip trip = dataRepo.getTripById(tripId);
                                 if (trip != null) {
                                     final String shapeId = trip.getShapeId();
                                     if (shapeId != null) {
                                         final Map<Integer, ShapePoint> shape = dataRepo.getShapeById(shapeId);
                                         if (shape != null) {
-                                            if (stopTime.getShapeDistTraveled() >
-                                                    distanceCalculationUtils
-                                                            .getShapeTotalDistance(shape, DistanceUnit.KILOMETER)) {
+                                            final float shapeTotalDistance = (float) distanceCalculationUtils
+                                                    .getShapeTotalDistance(shape, DistanceUnit.KILOMETER);
+                                            if (shapeDistanceTraveled > shapeTotalDistance) {
                                                 resultRepo.addNotice(
                                                         new FloatFieldValueOutOfRangeNotice("stop_times.txt",
```

Comment:
I wouldn't think of this particular error being "out of range" as currently implemented, as there really isn't a "range". It would be more of just an invalid value (but see my comment above).